app/src/main/java/ca/klapstein/baudit/models/RemoteModel.java

```diff
@@ -50,27 +50,102 @@ private static JestDroidClient createBaseClient() {
      * <p>
      * TODO: implement
      *
-     * @param username {@code String}
-     * @return {@code true} if the username {@code String} representation is not already taken, otherwise {@code false}
+     * return {@code true} if the username {@code String} representation is not already taken, otherwise {@code false}
      */
```

Comment:
instead of a custom query could we not just use `getPatient` and `getCareProvider` and check if both queries return `null`? If both return `null` that means no account exists that has the given `Username`